code/game/gamemodes/changeling/powers/epinephrine.dm

```diff
@@ -19,10 +19,11 @@
 	user.SetParalysis(0)
 	user.SetStunned(0)
 	user.SetWeakened(0)
+	user.setStaminaLoss(0)
 	user.lying = FALSE
 	user.update_canmove()
-	user.reagents.add_reagent("synaptizine", 20)
-	user.adjustStaminaLoss(-75)
+	user.reagents.add_reagent("synaptizine", 15)
```

Comment:
is there any reason to keep the synaptizine? the stun reduction is so minor its basically inconsequential. it was added 7 years ago in PR 129.

Reply:
update, just looked at it. the stun reduction isn't actually that terrible. comment can be dismissed.